### PR TITLE
Topics page. Move around sub topics. "Improve your IA" module publish.

### PR DESCRIPTION
**Pull request by sirleech**

### CODE REVIEW

**Comment by Phend-117** [comment]
There's a problem with the redirects. Investigating now.

File: src/sass/base/vd-tweaks.scss [hunk 1]
```diff
@@ -38,20 +38,13 @@
 ///
 //////////////////////////////////////////////
 
+// line length--> re: Buttericks Typography
+// https://practicaltypography.com/line-length.html
 .au-body p {
 	max-width: 35rem;
```
Comment (alex-page) [inline src/sass/base/vd-tweaks.scss]: There is a variable in the design system for this. I would use that variable or override it.

https://github.com/govau/uikit/blob/master/packages/core/src/sass/_globals.scss#L737

File: src/sass/base/vd-tweaks.scss [hunk 2]
```diff
@@ -38,20 +38,13 @@
...
-.spacer__sm {
- height: 1.5rem;
-}
-
-.spacer__med {
-	height: 2rem;
-}
-
-
+// hack to remove the vanilla focus from the main menu
```
Comment (alex-page) [inline src/sass/base/vd-tweaks.scss]: This is more than just the main menu, this is any area with tabindex -1

File: src/sass/components/topics_cards.scss [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	@include AU-media( md ) {
+		max-width: 90%;
+	}
+}
+
+h2.topicscards__headline {
+	max-width: 90%;
+}
+
+.topicscards__viewall {
```
Comment (alex-page) [inline src/sass/components/topics_cards.scss]: You should start using the `AU-space` function which uses unit and has pixel fallbacks for older browsers.

File: src/text/homepagebannercards.js [hunk 4]
```diff
@@ -23,7 +23,7 @@ const HomepageBannerCards = ( page ) => {
 					</div>
 				</div>
 
-				<div className="row au-grids cards guideshomepagecards">
+				<div className="row cards guideshomepagecards">
```
Comment (alex-page) [inline src/text/homepagebannercards.js]: `guideshomepagecards` should probably be `cards--home` if we are trying to use BEM

**Pull request merged**